supermarq-benchmarks/supermarq/converters.py

```diff
@@ -4,7 +4,7 @@
 import qiskit
 
 
-def cirq_to_qiskit(circuit: cirq.Circuit) -> qiskit.circuit.QuantumCircuit:
+def cirq_to_qiskit(circuit: cirq.Circuit) -> qiskit.QuantumCircuit:
```

Comment:
why these changes? `qiskit.QuantumCircuit` should be ok

Reply:
It was previously `qiskit.circuit.QuantumCircuit` (and locally that was picking up as `Any` sometimes -- like highlighting docstrings) so I updated it to just `qiskit.QuantumCircuit` to match the typing used in `qiskit-superstaq/`